commando/README.md

```diff
@@ -0,0 +1,27 @@
+# Commando plugin
+
+This plugin allows other nodes to send your node commands, and allows you
+to send them to other nodes.  The nodes must be authorized, and must be
+directly connected.
+
+Motto: Reckless?  Try going commando!
+
+## Installation
+
+For general plugin installation instructions see the repos main
+[README.md](https://github.com/lightningd/plugins/blob/master/README.md#Installation)
+
+## Options:
+
+Each of these can be specified more than once:
+
+* --commando-reader: a node id which can execute `list` and `get` commands
+* --commando-writer: a node id which can execute any commands.
+
+## Example Usage
+
+$ l1-cli plugin start commando.py commando_reader=0266e4598d1d3c415f572a8488830b60f7e744ed9235eb0b1ba93283b315c03518
```

Comment:
@rustyrussell This does not work.

Reply:
Thanks, you're right.  It needs to be a keyword-based argument list, like so:

```
$ l1-cli plugin subcommand=start plugin=`pwd`/commando.py commando_reader=0266e4598d1d3c415f572a8488830b60f7e744ed9235eb0b1ba93283b315c03518
```

(And path to commando.py needs to be absolute).